src/librustc/mir/mod.rs

```diff
@@ -2066,7 +2066,7 @@ pub enum Rvalue<'tcx> {
     Use(Operand<'tcx>),
 
     /// [x; 32]
-    Repeat(Operand<'tcx>, u64),
+    Repeat(Operand<'tcx>, &'tcx ty::Const<'tcx>),
```

Comment:
Wondering now if this should be `Constant` or if that's somewhat pointless.

Reply:
It's guaranteed to be a `usize`, it could be `ConstKind` :laughing:

Reply:
> It's guaranteed to be a `usize`, it could be `ConstKind`

Interning is *good*.